src/prism-async.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import highlight from './highlight';
+
+const languages = [];
+
+export const registerLanguage = (_, language) => {
+  if(ReactHighlightAsync.refractor) {
+    return refractor.register(language);
+  } else {
+    languages.push(language);
+  }
+};
+
+export default class ReactHighlightAsync extends React.PureComponent {
+  static refractor = null;
+  static highlightInstance = (highlight(null, {}));
+  static refractorPromise = null;
+  
+  static _loadRefractor() {
```

Comment:
small nitpick but prefer to not have the leading underscore even for "private functions" 

i should update lint file when i have time

Reply:
I don't think there is a linter currently? maybe we should also add prettier?